_source/handbook/04_streams.md

```diff
@@ -186,6 +186,40 @@ Turbo Streams consciously restricts you to seven actions: append, prepend, (inse
 
 Embracing these constraints will keep you from turning individual responses in a jumble of behaviors that cannot be reused and which make the app hard to follow. The key benefit from Turbo Streams is the ability to reuse templates for initial rendering of a page through all subsequent updates.
 
+## Custom Actions
```

Comment:
@marcoroth I'd appreciate reviewing on these documentation improvements. 

If you only review one section, I'd love for it to be the `Custom Actions` one!